template/default.tmpl

```diff
@@ -67,7 +67,6 @@ AlertmanagerUrl:
 {{ define "wechat.default.agent_id" }}{{ end }}
```

Comment:
We still need default values for the new fields: `Title`, `Description`, `URL`, ...